functions/go/set-image/transformer/images_transformer.go

```diff
@@ -43,21 +48,21 @@ func (t SetImage) Run(ctx *fn.Context, functionConfig *fn.KubeObject, items fn.K
 	if err != nil {
 		ctx.ResultErrAndDie(err.Error(), nil)
 	}
-	err = t.validateInput()
+	err = t.validateInput(items)
```

Comment:
Suggest adding a sanity check to notify users if the `ResourceList.Items`  is empty. Like what we did in set-labels